web/src/components/RunDetailAutomateMethods/methods/CLICommand/Controls.tsx

```diff
@@ -19,55 +16,59 @@ const optionToText: Record<CliCommandOption, string> = {
   [CliCommandOption.useDocker]: 'Run CLI via Docker image',
 };
 
-const defaultOptions: TCliCommandEnabledOptions = {
-  [CliCommandOption.Wait]: false,
-  [CliCommandOption.UseHostname]: false,
-  [CliCommandOption.UseCurrentEnvironment]: true,
-  // [CliCommandOption.UseId]: true,
-  [CliCommandOption.GeneratesJUnit]: false,
-  [CliCommandOption.useDocker]: false,
-};
-
 interface IProps {
   onChange(cmdConfig: TCliCommandConfig): void;
+  test: Test;
+  environmentId?: string;
 }
 
-const Controls = ({onChange}: IProps) => {
-  const [enabledOptions, setEnabledOptions] = useState<Record<CliCommandOption, boolean>>(defaultOptions);
-  const [format, setFormat] = useState<CliCommandFormat>(CliCommandFormat.Pretty);
-
-  const onSwitchChange = useCallback((name: CliCommandOption, isEnabled: boolean) => {
-    setEnabledOptions(prev => ({...prev, [name]: isEnabled}));
-  }, []);
+const Controls = ({onChange, test, environmentId}: IProps) => {
```

Comment:
Even if it's bad for the moment, it is better to keep using a standard way of handling forms, we can update this to use a new way in the near future